release-lockfile.spec.md

```diff
@@ -150,8 +150,12 @@ way.
 
 ## Format
 
-The canonical format for the release lockfile is a JSON document containing a
-single JSON object.  
+The canonical format for the release lockfile JSON document containing a
+single JSON object.  Lockfiles **must** conform to the following serialization rules.
+
+* The document must be tighly packed, meaning no linebreaks or extra whitespace.
```

Comment:
"must" is the spec usage here, so should probably be **must**.
Why not just "all" whitespace instead of "extra"?
Also, I am guessing this means no newline at the end of the file, but better to be explicit.